spacewalk/uyuni-setup-reportdb/bin/uyuni-setup-reportdb

```diff
@@ -355,6 +370,20 @@ exists_schema() {
     return $?
 }
 
+get_rhnconf_value() {
```

Comment:
Is this `get_rhnconf_value` used somewhere?

Reply:
removed